src/schemapack/spec/schemapack.py

```diff
@@ -156,7 +142,14 @@ class Relation(FrozenBaseModel):
 class ClassDefinition(FrozenBaseModel):
     """A model for describing a schemapack class definition."""
 
-    id: IdField
+    id_property: str = Field(
+        description=(
+            "A name that can be used for the ID property. It may not collide"
+            + " with content or relations properties."
+            + "This name e.g. relavant for specifying the ID property in a"
+            + " denormalized representation."
```

Comment:
```suggestion
            "The property name of the ID property in denormalized representations. It may not collide with top level properties of the content schema or relations properties."
```
Is there another use case? This description is a bit hard to understand 😅

Reply:
It may have another use case, just wanted to keep it generic